src/services/schema/mod.rs

```diff
@@ -0,0 +1,22 @@
+use async_trait::async_trait;
+use cedar_policy::Schema as CedarSchema;
+use cedar_policy::SchemaError;
+
+use crate::schemas::schema::Schema as InternalSchema;
+
+pub mod memory;
+pub mod load_from_file;
+
+#[async_trait]
+pub trait SchemaStore: Send + Sync {
+    async fn schema(&self) -> CedarSchema;
```

Comment:
would rename this function name, it is pretty confusing which does what ( get_schema and schema )